sdk/trace/src/main/java/io/opentelemetry/sdk/trace/SdkTracerProvider.java

```diff
@@ -100,6 +103,25 @@ public Sampler getSampler() {
     return sharedState.getSampler();
   }
 
+  /**
+   * Updates the tracer configurator, which computes {@link TracerConfig} for each {@link
+   * InstrumentationScopeInfo}.
+   *
+   * <p>This method is experimental so not public. You may reflectively call it using {@link
+   * SdkTracerProviderUtil#setTracerConfigurator(SdkTracerProvider, ScopeConfigurator)}.
+   *
+   * @see TracerConfig#configuratorBuilder()
+   */
+  void setTracerConfigurator(ScopeConfigurator<TracerConfig> tracerConfigurator) {
```

Comment:
Updated naming to mirror `SdkTracerProviderBuilder`, added javadoc pointing to the SdkTracerProviderUtil method for updatingl